test/TensorFlow/crashers.swift

```diff
@@ -6,6 +6,7 @@ import TensorFlow
 
 var someGlobal = Tensor<Int32>(1)
 
+// expected-warning @+1 {{value implicitly copied to the host}}
```

Comment:
the changes in sends/recvs warnings are fairly distracting -- should i ignore all of them when reviewing this patch?

Ideally, we could send a patch to remove all such warnings first, as discussed.

Reply:
Just ignore these for now. The good thing is that the runtime tests pass, which gives me *some* confidence that the refactoring is still generating correct code. 

(I will send out a separate PR removing these send/receive warnings along with adding an internal counter for debugging/testing purposes.)

Reply:
How do we confirm that this patch has not overall regressed in the # of sends/recvs?